bin/templates/cordova/lib/build.js

```diff
@@ -249,8 +253,10 @@ class ElectronBuilder {
             APP_AUTHOR: packageJson.author,
             APP_ID: packageJson.name,
             APP_TITLE: packageJson.displayName,
-            APP_WWW_DIR: this.api.locations.www,
+            APP_ICON: 'installer.png',
```

Comment:
To remove any potential confusion, it might be better to rename this to `APP_INSTALLER_ICON`.

```suggestion
            APP_INSTALLER_ICON: 'installer.png',
```